pkg/framework/objects/service_token/data_source_acceptance_test.go

```diff
@@ -1,4 +1,4 @@
-package data_sources_test
+package service_token_test
```

Comment:
Like for the resource, can we add a test or update the existing one where we add some environment level restrictions?

Reply:
I will add. Can we check to make sure that the feature is enabled in the instance used by CI ?